lib/ansible/modules/storage/ceph/ceph_volume.py

```diff
@@ -0,0 +1,273 @@
+#!/usr/bin/python
+
+#
+# Copyright (c) 2018 Red Hat, Inc.
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: ceph_volume
+
+short_description: Create ceph OSDs with ceph-volume
```

Comment:
I think you should at least provide update volume functionality to provide idempotency.